test/integration/lmtp/inspec/lmtp_spec.rb

```diff
@@ -32,7 +32,7 @@
 end
 
 describe file '/var/log/maillog' do
-  its('content') { should match(/dovecot: lmtp\(foo\): .* saved mail to INBOX$/) }
+  its('content') { should match(/saved mail to INBOX$/) }
```

Comment:
Why was this changed?

Reply:
That Inspec test fails on EL8 if it's unchanged. I think the logging may have changed between versions, but I'm not sure.

Reply:
Does the log not show up in that file or does it show up somewhere else like in `journalctl`?

Reply:
There are some extraneous characters added into the log, and I'm not great at regex. Should be fixed with my latest commit.